AnkiDroid/src/test/java/com/ichi2/anki/DeckPickerTest.kt

```diff
@@ -500,7 +501,13 @@ class DeckPickerTest : RobolectricTest() {
         runTest {
             startActivityNormallyOpenCollectionWithIntent(DeckPicker::class.java, Intent()).run {
                 val didA = addDeck("Deck 1")
-                supportFragmentManager.selectContextMenuOption(DeckPickerContextMenuOption.CREATE_SHORTCUT, didA)
+                viewModel.flowOfCreateShortcut.test {
+                    supportFragmentManager.selectContextMenuOption(DeckPickerContextMenuOption.CREATE_SHORTCUT, didA)
+                    advanceUntilIdle()
+                    awaitItem()
+                }
```

Comment:
My comment was about the two lines below:

```suggestion
supportFragmentManager.selectContextMenuOption(DeckPickerContextMenuOption.CREATE_SHORTCUT, didA)
advanceUntilIdle()    
```

being enough for the test to pass. If we require _test {}_ then there's something wrong in the code.